.github/workflows/workflow.yml

```diff
@@ -152,120 +181,59 @@ jobs:
-
-  test-inclusion:
-    needs: [build]
-    runs-on: ubuntu-latest
+  
+  build-and-publish-docker-image:
+    needs: [pop-runner-instance, build]
+    runs-on: ${{ needs.pop-runner-instance.outputs.runner }}
+    env:
+      ACTIONS_ALLOW_UNSECURE_COMMANDS: true
+      ECR_URL: 601305236792.dkr.ecr.eu-west-1.amazonaws.com
+      AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
+      AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
     steps:
       - uses: actions/checkout@v2
         with:
           fetch-depth: 0
           ref: ${{ github.event.pull_request.head.sha }}
+      - name: sleep 60
+        run: sleep 60    
+      - name: Configure AWS CLI
+        run: |
+          sudo apt-get -y install awscli
+          aws configure set aws_access_key_id ${{ env.AWS_ACCESS_KEY_ID }}
+          aws configure set aws_secret_access_key ${{ env.AWS_SECRET_ACCESS_KEY }}
+          aws configure set default.region eu-west-1   
       - uses: actions/download-artifact@v2
         name: Download basilisk binary
         with:
           name: basilisk
-          path: target/release
-      - uses: robinraju/release-downloader@v1
-        name: Download relay chain binary
-        with:
-          repository: paritytech/polkadot
-          tag: v0.9.12
-          fileName: polkadot
-          out-file-path: ../polkadot/target/release
-      - run: chmod +x ./basilisk && ./basilisk --version
-        name: Basilisk version
-        working-directory: target/release
-      - run: chmod +x ./polkadot && ./polkadot --version
-        name: Polkadot version
-        working-directory: ../polkadot/target/release
-      - uses: actions/setup-node@v2
-        name: Setup node.js
-        with:
-          node-version: '14'
-          check-latest: true
-      - run: ./install_polkadot_launch.sh
-        name: Setup polkadot launch
-        working-directory: scripts
-      - run: npm install
-        name: Setup wait script
-        working-directory: scripts/wait-for-inclusion
-      - run: yarn start ../../rococo-local/config.json & cd ../wait-for-inclusion && npm start 200
-        name: Launch rococo local & wait for inclusion
-        working-directory: scripts/polkadot-launch
+          path: .
+      - name: Add execution for basilisk
+        run: chmod +x basilisk
+      - name: Sleep for 60 seconds
+        run: sleep 60
+      - name: install docker
+        run: sudo snap install docker
+      - name: Sleep for 30 seconds
+        run: sleep 30s
+      - name: build docker image
+        run: sudo docker build -t basilisk:latest .
+      - name: get ecr credentials
+        run: aws ecr get-login-password --region eu-west-1 | sudo docker login --username AWS --password-stdin ${{ env.ECR_URL }}
+      - name: push image to ecr repository
+        run: sudo docker tag basilisk:latest ${{ env.ECR_URL }}/basilisk
+      - run: sudo docker push ${{ env.ECR_URL }}/basilisk:latest
 
-  test-upgrade-runtime:
```

Comment:
why is this removed?

Reply:
was for test only, adding it before merge